presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -148,7 +148,8 @@ protected QueryRunner createQueryRunner()
                 postgreSqlServer,
                 ImmutableMap.of(),
                 ImmutableMap.of("jdbc-types-mapped-to-varchar", "Tsrange, Inet" /* make sure that types are compared case insensitively */),
-                ImmutableList.of());
+                ImmutableList.of(),
+                1);
```

Comment:
Using 2+ nodes here is redundant, but do we gain much by requesting 1 node here?
If not, let's keep the code simpler.

Reply:
We are not testing engine. Rather, by using distributed runner we are testing connector properly passing information between nodes -- within column handles, etc, whatever is necessary.

This is the premise for the check here
https://github.com/prestosql/presto/blob/aa6117c38bf9b9921d36a44615e7694a6f5faea0/presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java#L53

While I do not see a particular reason why type mapping tests _need_ that, I would rather be on the safe side.

> they are more convenient to use locally as their start up time is shorter

How much is this?